docs/source/examples/checks/integrity/data_duplicates.ipynb

```diff
@@ -5,19 +5,28 @@
    "id": "06bdcdfc",
```

Comment:
<p>The <code>DataDuplicates</code> check finds multiple instances of similar samples in the Dataset. Duplicate samples increase the weight the model gives to those samples. If these duplicates are there intentionally (e.g. as a result of intentional oversampling, or due to the dataset's nature that there are similar-looking samples) this may be valid, however if this is an hidden issue we're not expecting to occur, it may be an indicator for a problem in the data pipeline that requires attention.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>